contributors/_contributors.yaml

```diff
@@ -1221,4 +1221,10 @@ houssein:
   twitter: hdjirdeh
   github: housseindjirdeh
   glitch: housseindjirdeh
-  
\ No newline at end of file
+  
+mdiblasio:
+  name:
+    given: Michael
+    family: DiBlasio
+  github: mdiblasio
+  glitch: mdiblasio
```

Comment:
Add newline. I think you may also need to rebase this file.

Reply:
how do I rebase?

Reply:
Oh I can do it if you're not familiar with it.

Reply:
if you don't mind, thanks :)